nym-vpn-lib/src/platform/android.rs

```diff
@@ -133,12 +149,9 @@ pub extern "system" fn Java_net_nymtech_vpn_NymVpnService_runVPN(_env: JNIEnv, _
     let state = RUNTIME.block_on(get_vpn_state());
     if state != ClientState::Disconnected {
         warn!("Invalid vpn state: {:?}", state);
-        return;
```

Comment:
Why remove this `return`, when there's no reason to continue in a corrupted state?

Reply:
Clippy complained.

Reply:
Hmm, can you reproduce that clippy run? It doesn't complain if I run with it.

Reply:
I don't see that run anymore. Lets go with it if clippy is happy 😆